prowler/providers/aws/services/acm/acm_certificates_with_secure_key_algorithms/acm_certificates_with_secure_key_algorithms.metadata.json

```
@@ -1,31 +1,46 @@
-  "RelatedUrl": "https://docs.aws.amazon.com/acm/latest/userguide/acm-certificate.html",
+  "Description": "**ACM certificates** are evaluated for the **public key algorithm and size**, identifying those that use weak parameters such as `RSA-1024` or ECDSA `P-192`. Certificates using `RSA-2048+` or ECDSA `P-256+` meet the secure baseline.",
+  "Risk": "**Weak certificate keys** reduce TLS confidentiality and authenticity.\n\nFeasible factoring or discrete log attacks can reveal private keys, enabling **man-in-the-middle**, session decryption, and **certificate spoofing**, leading to data exposure and tampering.",
+  "RelatedUrl": "",
+  "AdditionalURLs": [
+    "https://noise.getoto.net/2022/11/08/how-to-evaluate-and-use-ecdsa-certificates-in-aws-certificate-manager/",
+    "https://registry.terraform.io/modules/tedilabs/domain/aws/latest/submodules/amazon-issued-cert",
+    "https://www.encryptionconsulting.com/difference-between-various-public-key-infrastructure/",
+    "https://stackoverflow.com/questions/67606544/how-to-solve-aws-cloudfront-ssl-certificate-doesnt-exist",
+    "https://repost.aws/questions/QUhtmyY4d5SK6dvKUen8A7xg/aws-acm-algorithm-and-encryption",
+    "https://sdk.amazonaws.com/kotlin/api/latest/acm/aws.sdk.kotlin.services.acm.model/-request-certificate-request/-builder/key-algorithm.html",
+    "https://docs.aws.amazon.com/acm/latest/userguide/acm-certificate.html",
+    "https://docs.aws.amazon.com/securityhub/latest/userguide/acm-controls.html#acm-2"
+  ],
   "Remediation": {
     "Code": {
       "CLI": "",
-      "NativeIaC": "",
-      "Other": "",
-      "Terraform": ""
+      "NativeIaC": "```yaml\n# CloudFormation: ACM certificate with secure key algorithm\nResources:\n  <example_resource_name>:\n    Type: AWS::CertificateManager::Certificate\n    Properties:\n      DomainName: <example_domain>\n      KeyAlgorithm: EC_prime256v1  # CRITICAL: ensures a secure key algorithm (ECDSA P-256) to pass the check\n```",
+      "Other": "1. In the AWS Console, go to Certificate Manager (ACM)\n2. Click Request a certificate and enter <example_domain>\n3. Under Key algorithm, select ECDSA P-256 (or RSA 2048)\n4. Complete validation (DNS is recommended)\n5. In the service using the certificate (e.g., ALB/CloudFront/API Gateway), replace the old certificate with the new one\n6. Delete the insecure certificate (e.g., RSA-1024 or P-192) once no longer in use\n7. For CloudFront, request the certificate in us-east-1",
+      "Terraform": "```hcl\n# Terraform: ACM certificate with secure key algorithm\nresource \"aws_acm_certificate\" \"<example_resource_name>\" {\n  domain_name   = \"<example_domain>\"\n  key_algorithm = \"EC_prime256v1\"  # CRITICAL: ensures a secure key algorithm (ECDSA P-256) to pass the check\n}\n```"
```

```suggestion
      "Terraform": "```hcl\n# Terraform: ACM certificate with secure key algorithm\nresource \"aws_acm_certificate\" \"<example_resource_name>\" {\n  domain_name   = \"<example_domain>\"\n  key_algorithm = \"EC_prime256v1\"  # CRITICAL: ensures a secure key algorithm (RSA-2048+ or ECDSA P-256+)\n}\n```"
```